python/pylibcudf/pylibcudf/tests/test_scalar.py

```diff
@@ -28,3 +28,46 @@ def test_from_py_notimplemented(val):
 def test_from_py_typeerror(val):
     with pytest.raises(TypeError):
         plc.Scalar.from_py(val)
+
+
+@pytest.mark.parametrize(
+    "np_type",
+    [
+        "bool_",
+        "str_",
+        "int8",
+        "int16",
+        "int32",
+        "int64",
+        "uint8",
+        "uint16",
+        "uint32",
+        "uint64",
+        "float32",
+        "float64",
+    ],
+)
+def test_from_numpy(np_type):
+    np = pytest.importorskip("numpy")
+    np_klass = getattr(np, np_type)
+    if np_type == "str_":
+        np_val = np_klass("1")
+    else:
+        np_val = np_klass(1)
```

Comment:
Seems like a one-liner to me
```suggestion
    np_val = np_klass("1" if np_type == "str_" else 1)
```

Reply:
It looks like you did everything else in the review, not sure if this was an intentional omission or an oversight.